outputs.tf

```
@@ -13,3 +13,7 @@ output "postman_files_bucket" {
 output "cloudwatch_log_group" {
   value = aws_cloudwatch_log_group.lambda_logs
 }
+
+output "lambda_security_group" {
+  value = aws_security_group.lambda_vpc_sg
```

You might need to do something like the following since the vpc_sg could be null and it's a list according to terraform
```suggestion
  value = var.vpc_subnet_ids != [] ? aws_security_group.lambda_vpc_sg[0] : null
```

---

Sounds good, will do.